### DISCUSSION TITLE
Espere a sincronizacion

### DISCUSSION THREAD

**Discussion by Magikkkz** [Chia-Network/chia-blockchain#7388] [Support]
hola, hace unos días que tengo mis plots de color amarillo, dice "espere sincronizacion" aun no figuran en verde y no se cual es el problema. espero puedan ayudarme

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
